### PR TITLE
Fix the logging path issue for the percona mysql logger.

### PR DESCRIPTION
**Pull request by yudaykiran**
**What this PR does / why we need it**:
Fix the logging path issue for the percona mysql logger.
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
Signed-off-by: Uday Kiran Y <uday.kiran@openebs.io>

### CODE REVIEW

File: e2e/ansible/playbooks/hyperconverged/test-k8s-percona-mysql-pod/k8s-percona-pod-prerequisites.yml [hunk 1]
```diff
@@ -29,6 +29,14 @@
...
   become: true
 
+- name: Get $HOME of K8s master for kubernetes user
+  shell: source ~/.profile; echo $HOME
+  args:
+    executable: /bin/bash
+  register: result_kube_home
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+  changed_when: true
+
```
Comment (ksatchit) [inline e2e/ansible/playbooks/hyperconverged/test-k8s-percona-mysql-pod/k8s-percona-pod-prerequisites.yml]: There is an identical task in the same pre-req file which uses "/tmp" as the location for storing files (that too is stored as result_kube_home). Can you freeze upon this

File: e2e/ansible/playbooks/hyperconverged/test-k8s-percona-mysql-pod/k8s-percona-pod-prerequisites.yml [hunk 2]
```diff
@@ -29,6 +29,14 @@
   delegate_to: "{{ groups['kubernetes-kubeminions'].0 }}"
   become: true
 
+- name: Get $HOME of K8s master for kubernetes user
+  shell: source ~/.profile; echo $HOME
+  args:
+    executable: /bin/bash
+  register: result_kube_home
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
```
Comment (kmova) [inline e2e/ansible/playbooks/hyperconverged/test-k8s-percona-mysql-pod/k8s-percona-pod-prerequisites.yml]: how will this work in a setup like GKE where there is no access to master?

Comment (yudaykiran) [reply e2e/ansible/playbooks/hyperconverged/test-k8s-percona-mysql-pod/k8s-percona-pod-prerequisites.yml]: In GKE the Kubemaster and Localhost will refer to the same IPs. All the logs are stored on the Localhost.

**Comment by yudaykiran** [comment]
@ksatchit , the reason for using the same name is due to the reason that the copy_task.yml and stern_task.yml, both use the result_home variable name and will not work if the variable name is changed. It might need a bulk change if we change this in these tasks.

**Pull request merged**